### PR TITLE
Feedback review

### PR DESCRIPTION
**Pull request by AndresCampuzano**
Includes:
* Bug fix - secondFace is showing without previous value selected from the user.
* Implement functional programming based on functions for each task.
* Add JS Docs.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -29,18 +29,6 @@ <h2>How did we do?</h2>
 
         <p>Please let us know how we did with your support request. All feedback is appreciated
             to help us improve our offering!</p>
-        <!-- <div class="buttons">
```
Comment (AndresCampuzano) [inline index.html:32]: Deleted unused code

Comment (AndresCampuzano) [inline index.html:32]: Deleted unused code

File: style.css [hunk 2]
```diff
@@ -163,8 +163,9 @@ input:checked + label{
...
-    color: white;
-    cursor: pointer;
-    }
\ No newline at end of file
+    form label:hover {
+        background-color: hsl(25, 97%, 53%);
+        color: white;
+        cursor: pointer;
+    }
+}
```
Comment (AndresCampuzano) [inline style.css:171]: Add missing }

File: index.html [hunk 3]
```diff
@@ -58,7 +46,7 @@ <h2>How did we do?</h2>
             <input type="radio" name="score" id="5" value="5">
             <label for="5">5</label>
 
-            <button onclick="submitbtn()" type="button" id="btn" class="btn">Submit</button>
+            <button onclick="handleSubmit()" type="button" id="btn" class="btn">Submit</button>
```
Comment (AndresCampuzano) [inline index.html:49]: Improve name

File: index.js [hunk 4]
```diff
@@ -1,21 +1,26 @@
...
     }
+}
 
-    var getSelectedValue = document.getElementById("score-btn").value;
-
-    if (getSelectedValue != null) {
-        document.getElementById("selected").innerHTML
-            = getSelectedValue.value;
-
+/**
```
Comment (AndresCampuzano) [inline index.js:16]: JS docs (they describe what's the purpose and behavior of the function)

File: index.js [hunk 5]
```diff
@@ -1,21 +1,26 @@
...
-
-function submitbtn() {
-    document.querySelector('.secondFace').style.display = 'block';
-    var getSelectedValue = document.querySelector(
-        'input[name="score"]:checked');
-
-    if (getSelectedValue != null) {
-        document.getElementById("selected").innerHTML
-            = getSelectedValue.value;
+/**
```
Comment (AndresCampuzano) [inline index.js:1]: JS Docs

File: index.js [hunk 6]
```diff
@@ -1,21 +1,26 @@
...
-
+/**
+ * Looks for the selected value,
+ * then triggers the secondFace section just if the user selects a value.
+ */
+const handleSubmit = () => {
+    const getSelectedValue = document.querySelector(
+        'input[name="score"]:checked');
+    if (getSelectedValue) {
+        renderSecondFade(getSelectedValue)
```
Comment (AndresCampuzano) [inline index.js:24]: Function separated into small functions to improve code readability and maintainability.

File: index.js [hunk 7]
```diff
@@ -1,21 +1,26 @@
...
-        document.getElementById("selected").innerHTML
-            = getSelectedValue.value;
+/**
+ * Shows in DOM the secondFace section, along with the selected value.
+ * @param {HTMLInputElement} selectedValue
+ */
+const renderSecondFade = (selectedValue) => {
+    // Looks for elements in DOM before changing them.
+    const secondFace = document.querySelector('.secondFace');
+    const selectedSpanText = document.querySelector("#selected");
```
Comment (AndresCampuzano) [inline index.js:8]: Since we're using querySelector, don't use getElementById to keep code consistency.

File: .gitignore [hunk 8]
```diff
@@ -0,0 +1,2 @@
+.idea
```
Comment (AndresCampuzano) [inline .gitignore:1]: Ignoring local files created by IntelliJ IDE. Nothing important to care about.

File: index.js [hunk 9]
```diff
@@ -1,21 +1,26 @@
...
+    // Checks that the secondFace and selectedSpanText exits in DOM.
+    if (secondFace && selectedSpanText) {
+        secondFace.style.display = 'block';
+        selectedSpanText.textContent = selectedValue.value;
     }
+}
 
-    var getSelectedValue = document.getElementById("score-btn").value;
```
Comment (AndresCampuzano) [inline index.js:13]: Do not declare variables with `var`, use `const` or `let`

File: index.js [hunk 10]
```diff
@@ -1,21 +1,26 @@
...
-
-    if (getSelectedValue != null) {
-        document.getElementById("selected").innerHTML
-            = getSelectedValue.value;
-
+/**
+ * Looks for the selected value,
+ * then triggers the secondFace section just if the user selects a value.
+ */
+const handleSubmit = () => {
```
Comment (AndresCampuzano) [inline index.js:20]: Use arrow functions instead of traditional functions.

**Review by Andre-desing — APPROVED** [review]
Thanks a lot, man, appreciate it! I've learned new things by investigating about the changes you made. I will definitely apply all of these to the code I make from now on.

**Pull request merged**